CustomCraftSML/Fabricators/CustomFabricatorBuildable.cs

```diff
@@ -0,0 +1,101 @@
+﻿namespace CustomCraft2SML.Fabricators
+{
+    using System;
+    using System.Reflection;
+    using Common;
+    using CustomCraft2SML.Serialization.Entries;
+    using SMLHelper.V2.Assets;
+    using SMLHelper.V2.Crafting;
+    using UnityEngine;
+
+    internal class CustomFabricatorBuildable : Buildable
+    {
+        protected readonly CustomFabricator FabricatorDetails;
+
+        public CustomFabricatorBuildable(CustomFabricator customFabricator)
+            : base(customFabricator.ItemID, customFabricator.DisplayName, customFabricator.Tooltip)
+        {
+            FabricatorDetails = customFabricator;
+            OnStartedPatching += FabricatorDetails.StartCustomCraftingTree;
+            OnFinishedPatching += FabricatorDetails.FinishCustomCraftingTree;
+        }
+
+        public override TechGroup GroupForPDA { get; } = TechGroup.Miscellaneous;
+        public override TechCategory CategoryForPDA { get; } = TechCategory.Misc;
```

Comment:
Update these to reflect the chosen PDA categories in `FabricatorDetails`.  
Set these values as the defaults.